docs/competition/teams/role-descriptions/reception.md

```diff
@@ -14,7 +14,7 @@ tickets and/or wristbands. Each reception staffer has a desk, chair, and
 computer.
 
 There are four flavours of visitor to the competition: competitors,
-team-leaders, volunteers and true-visitors. **Everyone** must complete a media consent form (MCF) OR hold a valid ticket (which also counts as an MCF). Under 18's must have this form completed by a parent or guardian.
+team-leaders, volunteers and true-visitors. **Everyone** must complete a media consent form (MCF) OR hold a valid ticket (which also counts as an MCF). Under 16's must have this form completed by a parent or guardian.
```

Comment:
This is not necessarily the case. Can we add a note here to find the teams coordinator if someone denies media consent.

Reply:
Fixed in 75bd460